docs/.wordlist.txt

```diff
@@ -11,10 +11,12 @@ Juju
 Kubernetes
 lifecycle
 MicroK
+mkdir
 Multus
 Observability
 ONOS
 ONF's
+ubuntu
```

Comment:
You probably won't need these 2 new words if you have the terminal commands inside `single quotes`

Reply:
Indeed :) Thanks!